v0/destinations/kustomer/data/KustomerIdentify.json

```diff
@@ -85,7 +85,45 @@
   },
   {
     "destKey": "birthdayAt",
-    "sourceKeys": ["traits.dob", "context.traits.dob", "traits.birthday", "context.traits.birthday"],
+    "sourceKeys": [
+      "traits.dob",
+      "context.traits.dob",
+      "traits.birthday",
+      "context.traits.birthday"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "custom",
+    "sourceKeys": ["traits.customObject", "context.traits.customObject"],
     "required": false
+  },
+  {
+    "destKey": "sentiment",
+    "sourceKeys": ["traits.sentiment", "context.traits.sentiment"]
+  },
+  {
+    "destKey": "whatsapps",
+    "sourceKeys": ["traits.whatsapps", "context.traits.whatsapps"]
+  },
+  {
+    "destKey": "facebookIds",
+    "sourceKeys": ["traits.facebookIds", "context.traits.facebookIds"]
+  },
+  {
+    "destKey": "instagramIds",
+    "sourceKeys": ["traits.instagramIds", "context.traits.instagramIds"]
+  },
+  {
+    "destKey": "socials",
+    "sourceKeys": ["traits.socials", "context.traits.socials"]
+  },
+  {
+    "destKey": "timeZone",
+    "sourceKeys": ["traits.timeZone", "context.traits.timeZone"]
+  },
+  {
+    "destKey": "defaultLang",
+    "sourceKeys": ["traits.defaultLanguage", "context.traits.defaultLanguage"]
   }
 ]
```

Comment:
We are missing these following attributes:

- externalIds
- sharedExternalids
- sharedEmails
- sharedPhones
- sharedSocials
- urls
- locations
- importedAt
- rev

Reply:
We are supporting 
- externalIds
- urls
- locations
already via `transform.js`

added support for `importedAt` and `rev` 

lets now add the shared fields right now. As we are not sure of the use case, I was not able to verify due to account limitations

Reply:
Sorry my bad I didn't look into the `transform.js`. Okay, then we can update these later as per user requirements.